FiveCalls/Podfile

```diff
@@ -10,6 +10,8 @@ end
 
 target 'FiveCalls' do
   common_pods
+  pod 'Auth0', '~> 1.9'
+  pod 'Kingfisher', '~> 4.0'
```

Comment:
Now that we have `Kingfisher` (which is 👌 ), can you remove `RemoteImageView` and set those images with Kingfisher instead?  There is a race condition with my RemoteImageView that I never bothered fixing, which causes the wrong image to sometimes be displayed on contact rows.